repositories/httpmodels/http_opensanctions_dataset_freshness.go

```diff
@@ -57,16 +59,18 @@ func AdaptOpenSanctionsLocalDatasetFreshness(datasets HTTPOpenSanctionsLocalData
 
 	for _, ds := range datasets.Datasets {
 		if ds.Name == "default" {
-			version = &ds.IndexVersion
+			version = ds.IndexVersion
 		}
 	}
 
 	if version == nil {
-		return models.OpenSanctionsDatasetFreshness{}, errors.New(
-			"could not find upstream default dataset")
+		return models.OpenSanctionsDatasetFreshness{},
+			errors.New("could not find upstream default dataset")
 	}
 
-	lastUpdatedAt, err := time.ParseInLocation("20060102150405", (*version)[:len(*version)-4], time.UTC)
```

Comment:
This logic is still unsound if we cannot rely on OpenSanctions version formats. They said the outliers in formatting were due to them not being updated in a long time, which _should_ not apply to the `default` dataset (which we are checking here), but it still raises the question of whether we should change the logic.